pyvista/utilities/helpers.py

```diff
@@ -940,3 +941,71 @@ def axis_rotation(points, angle, inplace=False, deg=True, axis='z'):
 
     rot_mat = transformations.axis_angle_rotation(axis_to_vec[axis], angle, deg=deg)
     return transformations.apply_transformation_to_points(rot_mat, points, inplace=inplace)
+
+
+def cubemap(path='', prefix='', ext='.jpg'):
+    """Construct a cubemap from 6 images.
+
+    Each of the 6 images must be in the following format:
+
+    - <prefix>negx<ext>
+    - <prefix>negy<ext>
+    - <prefix>negz<ext>
+    - <prefix>posx<ext>
+    - <prefix>posy<ext>
+    - <prefix>posz<ext>
+
+    Prefix may be empty, and extension will default to ``'.jpg'``
+
+    For example, if you have 6 images with the skybox2 prefix:
+
+    - ``'skybox2-negx.jpg'``
+    - ``'skybox2-negy.jpg'``
+    - ``'skybox2-negz.jpg'``
+    - ``'skybox2-posx.jpg'``
+    - ``'skybox2-posy.jpg'``
+    - ``'skybox2-posz.jpg'``
+
+    Parameters
+    ----------
+    prefix : str, optional
+        Prefix to the filename.
+
+    ext : str, optional
+        The filename extension.  For example ``'.jpg'``
+
+    path : str, optional
+        Directory containing the skybox images.
+
+    Returns
+    -------
```

Comment:
```suggestion
    Returns
    -------
    texture : pv.Texture
            Cubemap.
```

Reply:
Sorry, had just changed this a second ago without committing and pushing.

Reply:
No problem. It's my presure to see the exciting PR.